stdlib/functools.pyi

```diff
@@ -131,3 +135,19 @@ if sys.version_info >= (3, 8):
 
 if sys.version_info >= (3, 9):
     def cache(__user_function: Callable[..., _T]) -> _lru_cache_wrapper[_T]: ...
+
+class _HashedSeq(Generic[_T]):
```

Comment:
Maybe this should inherit from `list`:

```suggestion
class _HashedSeq(Generic[_T], list[_T]):
```

Reply:
Yup!

Reply:
done!

Reply:
Add `# type: ignore` to the end of the erroring line. This `__hash__` is intentionally different from `list`'s `__hash__`.

Reply:
Do i need to change `__hash__` to :

```python
def __hash__(self) -> Union[int, List[_T]], : ...

or 

def __hash__(self) -> Any: ...
```

or just revert these changes back to:

```python
class _HashedSeq(Generic[_T]):
```